x-pack/plugins/actions/docs/openapi/components/schemas/run_connector_subaction_postmessage.yaml

```diff
@@ -0,0 +1,37 @@
+title: The postMessage subaction
+type: object
+description: >
+  Test an action that sends a message to Slack.
+  It is applicable only when the connector type is `.slack_api`.
+required:
+  - subAction
+  - subActionParams
+properties: 
+  subAction:
+    type: string
+    description: The action to test.
+    enum:
+      - postMessage
+  subActionParams:
+    type: object
+    description: The set of configuration properties for the action.
+    properties:
+      channelIds:
+        type: array
+        maxItems: 1
+        description: >
+          The Slack channel identifier, which must be one of the `allowedChannels` in the connector configuration.
+        items:
+          type: string
+      channels:
+        type: array
+        description: >
+          The name of a channel that your Slack app has access to.
+        maxItems: 1
+        items:
+          type: string
```

Comment:
this is deprecated, I think we should remove it.

Reply:
I've added `deprecated:true`. In general, we don't remove these details from the docs when they're deprecated--only when they're discontinued/removed from the product.